src/assets/scripts/client/parsers/scopeCommandParser/ScopeCommandModel.js

```diff
@@ -0,0 +1,66 @@
+import _first from 'lodash/first';
+import _has from 'lodash/has';
+import _last from 'lodash/last';
+import _without from 'lodash/without';
+import _values from 'lodash/values';
+import {
+    IMPLIED_COMMANDS,
+    EXPLICIT_COMMANDS,
+    COMMAND_FUNCTIONS
+} from './scopeCommandMap';
+
+// FIXME: Remove dummy sector codes
+const SECTOR_HANDOFF_CODES = ['18', '19', '10', '12'];
+
+/**
+ * A command to be interpreted by the scope itself
+ * Used for manipulation of the display and other ATC tasks
+ *
+ * @class ScopeCommand
+ */
+export default class ScopeCommand {
+    constructor(command) {
+        this.aircraftReference = '';
```

Comment:
could you add doc blocks for these props too?

Reply:
Done